### ISSUE TITLE
snap install - dpkg error while running script

### ISSUE THREAD

**Issue by whopavan** [realpvn/easy-apache#8]
When installed through snap we get an error "dpkg: error: error opening configuration directory '/etc/dpkg/dpkg.cfg.d': Permission denied". Probably some missing permissions

![easy-apache-error](https://user-images.githubusercontent.com/52735646/95024083-8b80b700-069e-11eb-8fb3-2f6e18d4dda6.png)

**Comment by whopavan** [comment]
Update on this issue

Looks like we cannot check other packages status when installed through snap since the snap is sandboxed and it cannot access anything outside it
More about this in this forum - https://forum.snapcraft.io/t/my-snap-when-installed-says-dpkg-error-permission-denied/20381

Might have to stop building for snap or find a way to do this

**Comment by whopavan** [comment]
The only way to fix this issue would be
```
One alternative is if you wanted to ship a snap that ships a version of Apache configured
to run in the strict confinement sandbox that your script can manage. One benefit
of this is that it would run on any system snaps support rather than just Debian/Ubuntu derived ones.
```

**Comment by whopavan** [comment]
Couldn't find any help on that, closing :(

**Comment by desrod** [comment]
I recently ran into this attempt to run `cvescan` on a clean Ubuntu 14.04 (Trusty) template I deployed. 

I received the same error, and resolved it by doing `snap install cvescan --devmode` which does allow reading outside of its own sandbox, and that worked. 

Of course, there are [specific considerations](https://snapcraft.io/docs/snap-confinement) when using the 'devmode' confinement, so choose carefully.
